src/development/LocalTest/Services/Events/Implementation/EventsRepository.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Events.Repository;
+using Altinn.Platform.Events.Models;
+
+using LocalTest.Configuration;
+using LocalTest.Helpers;
+using Microsoft.Extensions.Options;
+
+using Newtonsoft.Json;
+
+namespace LocalTest.Services.Events.Implementation
+{
+    /// <summary>
+    /// Represents an implementation of <see cref="IEventsRepository"/> that will
+    /// read and write events to the file system.
+    /// </summary>
+    public class EventsRepository : IEventsRepository
+    {
+        private readonly LocalPlatformSettings _localPlatformSettings;
+
+        /// <summary>
+        /// Initialize a new instance of the <see cref="EventsRepository"/> class with the given settings.
+        /// </summary>
+        /// <param name="localPlatformSettings"></param>
+        public EventsRepository(IOptions<LocalPlatformSettings> localPlatformSettings)
+        {
+            _localPlatformSettings = localPlatformSettings.Value;
+        }
+
+        /// <inheritdoc />
+        public async Task<string> Create(CloudEvent cloudEvent)
+        {
+            string eventsFolder = GetEventsCollectionFolder();
+            Directory.CreateDirectory(eventsFolder);
+
+            string filePath = Path.Combine(eventsFolder, cloudEvent.Id.AsFileName());
+
+            string serializedCloudEvent = JsonConvert.SerializeObject(cloudEvent);
```

Comment:
Setting the time property is handled in cosmos, but I'm guessing for localTest this would be the best place to assign the time property.

Reply:
I agree. Adding a line for setting time to UtcNow.